code/game/turfs/closed/minerals.dm

```diff
@@ -17,10 +15,10 @@
 	plane = GAME_PLANE_UPPER
 	base_icon_state = "smoothrocks"
 
-	base_pixel_x = MINERAL_WALL_OFFSET
-	base_pixel_y = MINERAL_WALL_OFFSET
-	pixel_x = MAP_SWITCH(MINERAL_WALL_OFFSET, 0)
-	pixel_y = MAP_SWITCH(MINERAL_WALL_OFFSET, 0)
+	// This is static
+	// Done like this to avoid needing to make it dynamic and save cpu time
+	// 4 to the left, 4 down
+	transform = TRANSLATE_MATRIX(-4, -4)
```

Comment:
Does this show up in the map editor (BYOND's or otherwise)? pixel_x and pixel_y were making this look bad

Reply:
it will show up in 515 dm, I can use the compile time macro if you'd like